rollup/internal/controller/sender/estimategas.go

```diff
@@ -33,15 +34,19 @@ func (s *Sender) estimateLegacyGas(to *common.Address, value *big.Int, data []by
 	}, nil
 }
 
-func (s *Sender) estimateDynamicGas(to *common.Address, value *big.Int, data []byte, fallbackGasLimit uint64, baseFee uint64) (*FeeData, error) {
+func (s *Sender) estimateDynamicGas(to *common.Address, data []byte, baseFee uint64, fallbackGasLimit uint64) (*FeeData, error) {
 	gasTipCap, err := s.client.SuggestGasTipCap(s.ctx)
 	if err != nil {
 		log.Error("estimateDynamicGas SuggestGasTipCap failure", "error", err)
 		return nil, err
 	}
 
-	gasFeeCap := new(big.Int).Add(gasTipCap, new(big.Int).Mul(new(big.Int).SetUint64(baseFee), big.NewInt(2)))
-	gasLimit, accessList, err := s.estimateGasLimit(to, data, nil, gasTipCap, gasFeeCap, value, true)
+	// gasFeeCap = baseFee * 2 + gasTipCap
+	gasFeeCap := new(big.Int).SetUint64(baseFee)
```

Comment:
```go
gasFeeCap := getGasFeeCap(new(big.Int).SetUint64(baseFee), gasTipCap)
```

Reply:
fixed in [1f8c930](https://github.com/scroll-tech/scroll/pull/1183/commits/1f8c9308faa91ff64a6bca162732c01164aa5d28)